include/ftl/thread_abstraction.h

```diff
@@ -342,13 +342,15 @@ inline void CloseEvent(EventType const eventId) {
 inline void WaitForEvent(EventType &eventId, uint32 milliseconds) {
 	pthread_mutex_lock(&eventId.mutex);
 
+	constexpr uint32 mills_in_sec = 1000;
```

Comment:
Nice! Can we format the name to kMillisInSec

Reply:
So following from this: https://github.com/RichieSams/FiberTaskingLib/pull/82#discussion_r240081342, all constants should be named this way.